docs/react.md

```
@@ -71,6 +71,34 @@ const { channel } = useChannel('your-channel-name', (message) => {
+
+```jsx
+<ChannelProvider channelName="outer">
+  <ChannelProvider channelName="inner">
+    {/* useChannel(...) resolves to "inner"; useChannel("outer", ...) resolves to "outer" */}
+  </ChannelProvider>
+</ChannelProvider>
+```
+
+> [!NOTE]
+> Channel resolution is scoped by `ablyId`. A hook called without an `ablyId` uses the `default` one, so it only resolves to a `ChannelProvider` that also uses the `default` `ablyId`. If your `ChannelProvider` uses a non-default `ablyId`, you must pass it to the hook for the channel to be inferred, for example `useChannel({ ablyId: 'your-ably-id' })`.
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)